src/commands/bot/commandsource.js

```diff
@@ -0,0 +1,32 @@
+const { Command, CommandError, SwitchbladeEmbed, GitUtils } = require('../../')
+const { sep } = require('path')
+
+const REPOSITORY_URL = 'https://github.com/SwitchbladeBot/switchblade'
+
+module.exports = class CommandSource extends Command {
+  constructor (client) {
+    super({
+      name: 'commandsource',
+      aliases: ['cmdsource', 'source'],
+      category: 'bot',
+      parameters: [{
+        type: 'command',
+        full: true,
+        required: true,
+        missingError: 'commands:commandsource.missingCommand',
+        acceptHidden: true
+      }]
+    }, client)
+  }
+
+  async run ({ channel, author, t }, command) {
+    const branchOrHash = await GitUtils.getHashOrBranch()
+    if (branchOrHash === null) throw new CommandError(t('commands:commandsource.noRepositoryOrHEAD'))
+    if (!branchOrHash) throw new CommandError(t('commands:commandsource.branchNotUpToDate'))
+
+    channel.send(new SwitchbladeEmbed(author)
+      .setTitle(command.name)
```

Comment:
```suggestion
      .setTitle(command.fullName)
```

![Discord_FMXIXNxvwF](https://user-images.githubusercontent.com/25179120/93027993-2974ea80-f5e7-11ea-90a1-7e7cdba8e7d1.jpg)